monitoring/soca_plotfield.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+#Command:
+#   bash soca_plotfield.sh exp_name state_vector start_date num_of_days
+
+
```

Comment:
Same as other comments ... Go through that file and remove the debugging comments